ex_cubic_ods_ingestion/lib/ex_cubic_ods_ingestion/schema/cubic_ods_load.ex

```diff
@@ -31,4 +35,35 @@ defmodule ExCubicOdsIngestion.Schema.CubicOdsLoad do
 
     timestamps(type: :utc_datetime)
   end
+
+  @spec insert_from_objects(list()) :: tuple()
+  def insert_from_objects(objects) do
+    Repo.transaction(fn -> Enum.map(objects, &insert_ready(&1)) end)
+  end
+
+  @spec insert_ready(map()) :: Ecto.Schema.t()
+  def insert_ready(object) do
+    {:ok, last_modified, _offset} = DateTime.from_iso8601(object[:last_modified])
+    last_modified = DateTime.truncate(last_modified, :second)
+    size = String.to_integer(object[:size])
+
+    Repo.insert!(%__MODULE__{
+      status: "ready",
+      s3_key: object[:key],
+      s3_modified: last_modified,
+      s3_size: size
+    })
+  end
+
+  @spec get_s3_modified_since(DateTime.t()) :: [__MODULE__.t()]
```

Comment:
nitpick: you don't need the `__MODULE__` bit when referring to things in the current module.
```suggestion
  @spec get_s3_modified_since(DateTime.t()) :: [t()]
```